gluoncv/data/tracking_data/track.py

```diff
@@ -0,0 +1,537 @@
+""" siamrpn dataloader,include youtube-bb,VID,DET,COCO dataset
+Code adapted from https://github.com/STVIR/pysot """
+# coding: utf-8
+# pylint: disable=missing-docstring,unused-argument,arguments-differ
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import json
+import logging
+import os
+import numpy as np
+
+from mxnet.gluon.data import dataset
+from gluoncv.utils.filesystem import try_import_cv2
+from gluoncv.model_zoo.siamrpn.siamrpn_tracker import corner2center, center2corner
+from gluoncv.model_zoo.siamrpn.siamrpn_tracker import Center, Anchors
+from gluoncv.data.transforms.track import Augmentation
+from gluoncv.utils.metrics.tracking import Iou
+
+class SubDataset(object):
+    """Load the dataset for tracking.
```

Comment:
The description here is the same with TrkDataset, try to differentiate these two class. Otherwise users will confuse.